src/openrct2/paint/sprite/peep.c

```diff
@@ -76,8 +76,8 @@ void peep_paint(rct_peep * peep, int imageDirection)
 		spriteType = peep->next_action_sprite_type;
 		imageOffset = 0;
 	}
-
-	uint32 baseImageId = (imageDirection >> 3) + sprite.sprite_image[spriteType].base_image + imageOffset * 4;
+	
```

Comment:
It's a Hodge podge of formatting right now the whole PR as I had limited time. I wouldn't bother looking at it just yet